Versions.md

```diff
@@ -0,0 +1,23 @@
+# Versions
+
+The Microsoft Graph SDKs MUST follow [Semantic Versioning 2.0.0](https://semver.org/). This is important for capturing information about the use of each SDK via the SdkVersion header emitted by our libraries. 
+
+Changes made to the metadata and our generation tools that break a signature MUST result in a major version change.
+
+Changes made to the underlying API behavior of the service API that are not intentional should be considered a service bug and not result in a major version bump. Ideally we block the change from being propagated to the generated service libraries and the service API gets fixed. 
+
+## Version format per language
+
+Unless oherwise specified, the SDK library and packages should use the following format for specifying preview versions: `x.y.z-preview.n`.  An example of this is `1.3.0-preview.2`. The first preview release should be labeled like `1.3.0-preview`. Subsequent iterations of a preview can be suffixed with the preview iteration like this: `1.3.0-preview.1`. 
+
+Service libraries for the beta endpoints will not increment the pre-release version when new releases are made. We will typically increment the minor version across new beta service libraries `4.3.0-preview` --> `4.4.0-preview`.
+
+Prelease identifiers like `preview`, `beta`, and `rc` should be lower case.
```

Comment:
@Ndiritu @baywet What's your thoughts on prerelease identifiers being lower case? I'd like to standardize this so that data pipelines or analysis that uses prerelease identifiers don't need to account for case.

Reply:
Fine with me. Also tested that they work with Composer.
The PHP v2 preview naming already goes against these guidelines. Is it ok if we kept uppercase `RC` for v2?

Reply:
For now keep it for v2, we may change it based on this PR.